README.md

```diff
@@ -20,6 +20,8 @@ Other than tracing, Tracee is also capable of capturing files written to disk or
 To run, Tracee requires the following:
 - Linux kernel version > 4.14
 - Kernel headers
+
+In addition, if running the binary directly (not via docker):
 - C standard library (currently tested with glibc)
 - clang (currently tested with version > 10)
```

Comment:
we need to test which minimum version is required. if you `apt-get install clang` the stable release in debian gets you clang7. and I think it works

Reply:
Version 6 didn't work for me - So 7 it is